### DISCUSSION TITLE
Android Versions for Slingerplayer

### DISCUSSION THREAD

**Discussion by remgr** [GerryDazoo/Slinger#543] [Q&A]
Hi Gerry,

I am thinking of getting an older phone as a streaming device. I much prefer your apk, it is much better than VLC.  The phones that have usb-c-to hdmi are the flagship versions of the manufacturers.  I want to buy an older flagship=cheaper.  Which versions of android will this run on.  I tried it on an old tablet running android 8, it runs, but it doesn't let me edit the http address.  Will this work for android 9, 10 etc.  It works fine on my Pixel 6, but that has the latest android version.

Thanks again!

**Comment by GerryDazoo** [comment]
It "should" work starting on Android 7 SDK level 21. But the only "old" phone I have to test on is a Samsung S-10. But I can make no guarantees about the suitability of any particular phone. 
What do you mean exactly when you say it won't let you edit? What model tablet is it? You are clicking on the URL button to make changes? Please post a screenshot but try the attached APK first.
Here's the latest and greatest APK.
[SlingerPlayer.zip](https://github.com/GerryDazoo/Slinger/files/10830982/SlingerPlayer.zip)

**Reply by remgr** [reply]
Hi Gerry,

The tablet is a Samsung Tab A 10.1.  It is probably 5 years old.  I am also attaching the software info page.  The second attachment is the app screen shot that show's that no keyboard appears when pressing the URL button, only a second dialog screen pops up.  I can't even get Paste to work, to copy the URL I have on the clipboard.    I tried installing the SlingerPlayer you attached.  The app doesn't open.  It freezes on the icon picture.  I have attached that screenshot as well.  

I much prefer your SlingerPlayer to VLC on my current phone.  It opens much faster, the screen resolution is a better fit, and the remote on the bottom is priceless.  I have been using an iPhone (with the Slingplayer app) to watch the Slingbox after they removed the Slingbox channel from the Roku.  I would be more then happy to jailbreak the phone if your app had an ios compatible version.   I am certainly willing to buy an old LG phone which is capable of HDMI Alt Mode.  But the android version will max out at 10.  As always, your help is greatly appreciated. 

![Android Version](https://user-images.githubusercontent.com/114819659/221379404-16d89867-769e-4e22-80bb-65a56e815d60.jpg)

![SlingerPlayer Original](https://user-images.githubusercontent.com/114819659/221379428-aa65d716-483a-4e50-b33c-82e5b9791161.jpg)

![SlingerPlayer New One](https://user-images.githubusercontent.com/114819659/221379455-a2d06b0d-6aab-4c57-8993-649ad406ef32.jpg)

**Reply by GerryDazoo** [reply]
You're running the wrong APK. This is the Firestick version. You need the Android version.
https://github.com/GerryDazoo/SlingerPlayer

This is what the Android version looks like.
![image](https://user-images.githubusercontent.com/102599361/221406205-1109a390-3d88-48bf-a273-b77b1163fc70.png)



PS. Are you running Slinger V4.0 on the same LAN? The list of available streams should have been auto-populated.

**Comment by remgr** [comment]
Hi Gerry,

I couldn't get to the input keyboard, or copy and paste the WAN address on the SlingerPlayer app, so nothing would populate.  I use it on my phone without a problem, but it runs on Android 13.  All I am getting now is a frozen screen, the app no longer opens.  I have deleted and reinstalled it several times.  I used the SlingerPlayer you link to above, and even went back to the one I downloaded two weeks ago. I don't think it's compatible with Android 8.  

I have now tested it on my husband's phone, which is on Android 10.  It works perfectly there. Just an FYI.

**Reply by GerryDazoo** [reply]
Unfortunately, I have no way to test this on older Android devices. It's not doing anything fancy. I'll take a look at code to see if there is anything I can do.

**Comment by regmar505** [comment]
I have a 10 years old Nexus 7 tablet with Android 6.0.1 and I am glad to say that SlingerPlayer works well, audio is perfectly synced though video is not as smooth as with MXPlayer, I wonder whether SlingerPlayer uses hardware acceleration. The server is a Pi 3B+ and the slingbox is not on the same LAN.

As somebody else reported, SlingerPlayer does not load on my firestick gen 2.

**Reply by GerryDazoo** [reply]
Good news/bad news I guess. What video setting are you using? I had tried to pick some generic defaults for the stream buffering to avoid people from trying to configure them manually and getting it wrong. For fun can you try  1280x720 with a bitrate of 5000 to see if that helps. The hardware acceleration is turn off because it was causing me mucho audio/video sync problems. Also, what type of slingbox are you using?

**Reply by regmar505** [reply]
Current settings: Resolution=15, Framerate=30, VideoBandwidth=2000, VideoSmoothness=63

I do not see any improvements with Resolution=12, VideoBandwidth=5000. Regarding hardware acceleration, is it something that an user could enable/disable in config.ini?  I have a Slingbox M2. Thanks for all your great work!

**Reply by sp222222** [reply]
if it runs on six I bet I can sideload it onto 5.1.1 if I ever make time to. works fine on version 9. I would dial down the bandwidth bigtime regmar 505 to around 1664 and go lower until picture quality improves.  maybe smoothness of 58 and dial that lower too each try.
